demo/webcam_demo.py

```diff
@@ -27,61 +27,61 @@
 
 def parse_args():
     parser = argparse.ArgumentParser()
-    parser.add_argument('--cam_id', type=str, default='0')
+    parser.add_argument('--cam-id', type=str, default='0')
```

Comment:
Does this affect docs?

Reply:
No. The modified version matches the docs.

Reply:
Updated the docs.